tests/system/action/motion/test_follow_recommendation.py

```diff
@@ -76,46 +76,6 @@ def test_follow_recommendation_correct(self) -> None:
         assert model.get("state_extension") == "test_test_test"
         assert model.get("last_modified", 0) >= check_time
 
-    def test_follow_recommendation_not_neighbour(self) -> None:
```

Comment:
Keep this test and fix `motion.follow_recommendation`: There, the check for the next states should still exist.

Reply:
Readded the test and modify follow_recommendation and set_state, that the test passes.